wiki/Tournaments/TSC/1/en.md

```diff
@@ -70,27 +138,61 @@ The Taiko Suiji Cup is run by various osu!taiko community members.
 - The match lobby will be created by the referee, and the team captains of both teams will be invited. The captain is then responsible for inviting the rest of their team. If the captain of a team is not online, the referee will invite any other player from that team, and that player will be responsible for inviting their team.
 - Teams have a 15 minute grace period to have at least 4 players available to play the match. If 15 minutes pass and a team does not have enough players to play the match, the other team will receive a win by default.
 - Once all players have joined the lobby, captains will be asked to !roll; the winner of the roll will have the first warmup, first ban, and first pick.
+- The Mappool will contain 16 maps. The format is as follows: 
+  - Mappool A will contain the following: 4 NoMod, 2 Hidden, 2 HardRock, 2 FreeMod. 
+  - Mappool B will contain the following: 2 Challenge Maps, 1 DoubleTime, 1 Easy Mod, 1 Hidden+Hard Rock, and the Tiebreaker. These maps are **not bannable**.
```

Comment:
`Challenge Maps` -> `Challenge`?

`Hidden+Hard Rock` -> `Hidden+HardRock`

Reply:
I missed this and only did the `fr.md`. That's why it didn't get applied